test/integration_test/basic_test.go

```diff
@@ -208,3 +221,70 @@ func BasicUpgrade(tc *TestCase) func(*testing.T) {
 		require.NoError(t, err)
 	}
 }
+
+func InstallWithTelemetry(tc *TestCase) func(*testing.T) {
+	return func(t *testing.T) {
+		if tc.ShouldSkip() {
+			t.Skip()
+		}
+
+		testSpec := tc.TestSpec(t)
+		cluster, ok := testSpec.(*op_corev1.StorageCluster)
+		require.True(t, ok)
+
+		testInstallWithTelemetry(t, cluster)
+	}
+}
+
+func testInstallWithTelemetry(t *testing.T, cluster *op_corev1.StorageCluster) {
+	secret := testutil.GetExpectedSecret(t, "/specs/pure-telemetry-cert.yaml")
```

Comment:
This should be `/testspec`, right?

Reply:
right, have not retested functional test after some changes, but will get to that.